catalog/dags/data_refresh/reporting.py

```diff
@@ -26,7 +26,7 @@ def report_record_difference(before: str, after: str, media_type: str, dag_id: s
     # '+' - number will always have a sign in front of it
     # ',' - number is comma separated
     # '.' - number is a float
-    message = f"""
+    message = f"""Data refresh for {media_type} complete! :tada:
 _Note: All values are row estimates and are not (but nearly) exact_
```

Comment:
```suggestion
    message = f"""
Data refresh for {media_type} complete! :tada:
_Note: All values are row estimates and are not (but nearly) exact_
```
The message should probably still have a new line at the beginning, as it does now. If you add the new line as in the suggestion, I think this PR is ready. Thank you for your contribution, @viktoriussuwandi !

Reply:
Anytime @obulat, thanks for your suggestion also !